jenkins-client-it-docker/src/test/java/com/offbytwo/jenkins/integration/NoExecutorStartedPluginManagerIT.java

```diff
@@ -33,12 +33,12 @@ public void getPluginsShouldReturn9ForJenkins20() {
     }
 
     @Test
-    public void getPluginsShouldReturn27ForJenkins1651() {
+    public void getPluginsShouldReturn28ForJenkins1651() {
         JenkinsVersion jv = jenkinsServer.getVersion();
         if (jv.isLessThan("1.651") && jv.isGreaterThan("1.651.3")) {
```

Comment:
@aidenkeating not really sure TBH. I didn't write this test. Just update the name of the test as I added a new plugin